cmake/utils.cmake

```diff
@@ -240,7 +240,7 @@ macro(override_gpu_arches GPU_ARCHES GPU_LANG GPU_SUPPORTED_ARCHES)
       endif()
 
       if (_SM)
-        set(_VIRT "")
+        set(_VIRT "-real")
```

Comment:
Could you add a reference and the context?

Reply:
done

Reply:
Great!